openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -9,18 +10,44 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
     order = pyblish.api.IntegratorOrder
     label = "Kitsu Note and Status"
     families = ["render", "kitsu"]
+
+    # status settings
     set_status_note = False
     note_status_shortname = "wfa"
 
+    # comment settings
+    CustomCommentTemplate = {}
+    CustomCommentTemplate["enabled"] = False
+    CustomCommentTemplate["comment_template"] = "{comment}"
+
+    def safe_format(self, msg, **kwargs):
+        def replace_missing(match):
+            value = kwargs.get(match.group(1), None)
+            if value is None:
+                self.log.warning(
+                    "Key `{}` was not found in instance.data "
+                    "and will be rendered as `` in the comment".format(
+                        match.group(1)
+                    )
+                )
+                return ""
+            else:
+                return str(value)
+
+        pattern = r"\{([^}]*)\}"
+        return re.sub(pattern, replace_missing, msg)
+
     def process(self, context):
         # Get comment text body
         publish_comment = context.data.get("comment")
         if not publish_comment:
             self.log.info("Comment is not set.")
 
-        self.log.debug("Comment is `{}`".format(publish_comment))
-
         for instance in context:
+            # Check if instance is a render by checking its family
+            if "render" not in instance.data["family"]:
```

Comment:
What if it is a `review`?

Reply:
Yeah it should be review as that's what should be uploaded to Kitsu. Fixed